### ISSUE TITLE
Speed property get zero after a few seconds

### ISSUE THREAD

**Issue by bluelemonade** [NativeScript/nativescript-geolocation#138]
Hello, 

First I thought the issue appear only inside my project, but it's the same behaviour inside the playground app.

I am using this configs:
```
 desiredAccuracy: 3,
          updateDistance: 0.1,
          updateTime: 1000,
          minimumUpdateTime: 100,
          maximumAge: 5000
```
works great for most props, but the speed gets lost after a few refreshs, maximum 2 seconds.

### Which platform(s) does your issue occur on?
 Android version 
Huawei P10

code available in the playground, https://play.nativescript.org/?template=play-vue&id=KkjUBO&v=4

### Please, tell us how to recreate the issue in as much detail as possible. 
after a few seconds I get no speed property, its stays to 0.

### Is there any code involved? 
only playground

**Comment by lini** [comment]
The geolocation plugin uses [android.location.LocationManager](https://developer.android.com/reference/android/location/LocationManager) to get the location data on Android. Whatever is returned from the native code will be sent to your code. I am not sure why the retuned spped is zero but can offer a couple of suggestions:
- speed/direction can be incorrect if you are in a building. Try testing outside and waiting a few moments until the device gets a proper GPS lock before noting the speed.
- the phone's GPS settings might affect the returned data. Make sure the device is set to `High Accuracy` so the app can get the data it needs.

I was not able to find a HiSilicon Kirin 960 device to test the playground app, but the ones I tested on showed the speed changing when I moved around (alternating between zero and 1-2).
